website/static/css/pages/remove-contributor-page.css

```diff
@@ -0,0 +1,4 @@
+.col-md-4 {
+    padding-top: 20px;
+    padding-right: 15px;
+}
```

Comment:
I don't find anything in the `webpack.*.config.js` files to suggest that we're pipelining CSS, and, in fact, I find variously code references to various CSS files under `static/` from inside templates:

```
website/templates/project/contributors.mako
171:<link rel="stylesheet" href="/static/css/pages/contributor-page.css">

website/templates/project/files.mako
19:        <link rel='stylesheet' href='${stylesheet}' type='text/css' />

website/templates/project/project.mako
397:    <link rel='stylesheet' href='${stylesheet}' type='text/css' />
400:    <link rel="stylesheet" href="/static/css/pages/project-page.css">

website/templates/project/settings.mako
405:    <link rel="stylesheet" href="/static/css/pages/project-page.css">

website/templates/public/login.mako
194:    <link rel="stylesheet" href="/static/css/pages/login-page.css">

website/templates/public/pages/active_nodes.mako
7:    <link rel="stylesheet" href="/static/css/pages/public-activity-page.css">

website/templates/public/pages/faq.mako
5:    <link rel="stylesheet" href="/static/css/pages/getting-started-page.css">

website/templates/public/pages/getting_started.mako
6:    <link rel="stylesheet" href="/static/css/pages/getting-started-page.css">

website/templates/public/pages/meeting.mako
10:    <link rel="stylesheet" href="/static/vendor/bower_components/hgrid/dist/hgrid.min.css" />

website/templates/public/pages/meeting_landing.mako
6:    <link rel="stylesheet" href="/static/css/pages/meeting-landing-page.css">

website/templates/public/pages/news.mako
4:    <link href='http://fonts.googleapis.com/css?family=Carrois+Gothic|Inika|Patua+One' rel='styleshe

website/templates/search.mako
5:    <link rel="stylesheet" href="/static/css/pages/search-page.css">
```